src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,94 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "Orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
```

Comment:
The `@OneToMany` annotation is used with a `@JoinTable`, which is not typical. Usually, `@OneToMany` is used with `@JoinColumn` or `mappedBy` for bidirectional relationships. Consider reviewing the relationship mapping to ensure it aligns with your database schema and application requirements.